drafts/bitcartel-payment-disclosure/draft.rst

```diff
@@ -0,0 +1,336 @@
+| Title: Payment Disclosure
+| ZIP: 00? (DRAFT)
+| Replaces: n/a
+| Superseded-by: n/a
+| Editor: Simon Liu <simon@z.cash>
+
+==================
+Payment Disclosure
+==================
+
+Abstract
+--------
+
+This ZIP describes a method of proving that a payment was sent to a shielded address.  In the typical case, this means enabling a sender to present a proof that they transferred funds to a recipient's shielded address.  The method described will be compatible with the Zcash Protocol [PROTOCOL] deployed at the launch of the Zcash network.
+
+This document follows the common conventions defined for ZIPs in [ZIP-1].
+
+.. [PROTOCOL] Zcash Protocol Specification: https://github.com/zcash/zips/blob/master/protocol/protocol.pdf
+.. [ZIP-1] https://github.com/zcash/zips/blob/master/zips/0001.rst
+
+Copyright Notice
+----------------
+This ZIP is published under the MIT License.
+
+Motivation
+----------
+Payment disclosure is useful in a number of situations, for example:
+
+* A sender may need to prove that their payment was sent and received by a recipient.  For example, a customer paid too much for an item and would like to claim a refund from the vendor.
+* A third party needs to verify that a payment between sender and recipient was executed successfully.  For example, a regulator needs to confirm a transfer of funds between two parties.
```

Comment:
This is broken when the sender and receiver collude (https://github.com/zcash/zcash/issues/2036#issuecomment-329864311).

The ZIP must be clearer on what are the claimed security properties of payment disclosure.

Reply:
What would you add here?  Payment disclosure proves a payment occurred between two parties, but it doesn't prove that the payment being disclosed is the actual payment a third party is interested in.

Reply:
The weaker property you describe seems true, but not trivially -- you will need to invoke the binding property of the commitment scheme to prove it. So don't believe it until you formally state the property and sketch the proof.

Reply:
With the `DecryptNote` fix, maybe the third party _will_ be convinced of the amount as well (checking its consistency with the commitment). Again, this needs a proof.

Reply:
Publishing this zip should not be blocked on a security proof; that should be in a ZIP similar to [310](https://zips.z.cash/zip-0310)